src/server/routes/actions.spec.ts

```diff
@@ -11,158 +12,115 @@ import request from "supertest";
 import { BotRequestType, FaucetRequestType } from "../../types";
 import router from "./actions";
 
-jest.mock("../services/ActionStorage");
-jest.mock("../services/Actions", () => {});
+jest.mock("../../dripper/DripperStorage");
+jest.mock("../../dripper/polkadot/PolkadotActions", () => {});
 jest.mock("../../logger", () => {
   return { logger: { error: mockLoggerError } };
 });
 
-jest.mock("../services/DripRequestHandler", () => {
+jest.mock("../../dripper/DripRequestHandler", () => {
   return {
+    getDripRequestHandlerInstance: jest.fn().mockImplementation(() => {
+      return { handleRequest: mockHandleRequest };
+    }),
```

Comment:
I didn't wanted to add a factory method, but I will say, I **really was tempted to do so**.